### ISSUE TITLE
[SB Track2] Do not close receive link after each sync receive

### ISSUE THREAD

**Issue by conniey** [Azure/azure-sdk-for-java#9786]
Do not close receive link after each receive operation on the synchronous client.

**Comment by conniey** [comment]
Depends on #9792 being fixed.

**Comment by hemanttanwar** [comment]
duplicate of another issue which is already fixed.
